### DISCUSSION TITLE
Any updates on card print numbers and buttons?

### DISCUSSION THREAD

**Discussion by psychosyko** [NoMeansNowastaken/KarutaSniper#18] [General]
I also would recommend to add an option for high wishlisted cards, idk if that is or not possible tho

**Comment by splat58** [comment]
ive been super busy recently and dont have free time anymore, but high whitelisted cards, while possible, would be hard to make because there is no easy way to get a list of all the cards (that i know of)

**Comment by NoMeansNowastaken** [comment]
upon second thought, this could be implemented, but in a no so accurate way
